modules/openapi-generator/src/main/resources/Java/libraries/native/README.mustache

```diff
@@ -94,7 +95,12 @@ public class {{{classname}}}Example {
         {{{dataType}}} {{{paramName}}} = {{{example}}}; // {{{dataType}}} | {{{description}}}
         {{/allParams}}
         try {
-            {{#returnType}}{{{returnType}}} result = {{/returnType}}apiInstance.{{{operationId}}}({{#allParams}}{{{paramName}}}{{#hasMore}}, {{/hasMore}}{{/allParams}});{{#returnType}}
+            {{^vendorExtensions.x-group-parameters}}
+            {{#returnType}}{{{returnType}}} result = {{/returnType}}apiInstance.{{{operationId}}}({{#allParams}}{{{paramName}}}{{#hasMore}}, {{/hasMore}}{{/allParams}});{{/vendorExtensions.x-group-parameters}}{{#vendorExtensions.x-group-parameters}}{{#hasParams}}
```

Comment:
It may add extra empty line for` {{^vendorExtensions.x-group-parameters}}`case

Reply:
IMHO having an extra empty line in auto generated source code is alright if it can make the maintenance easier. Also, I don't know how about the java clients but the ruby one has the option to run a linter afterwards to take care of such things.

Reply:
It won't result in empty lines if the start tag and end tag are aligned properly. Ref: https://github.com/samskivert/jmustache#newline-trimming